CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Library path (`DelphiLibraryPath`/`DelphiTranslatedLibraryPath`)
   - Browsing path (`DelphiBrowsingPath`)
   - Standard library
+- Empty anonymous methods are now ignored in `EmptyBlock`.
+- Empty anonymous methods are now flagged in `EmptyRoutine`.
```

Comment:
API additions to `AnonymousMethodNode` should be mentioned in the changelog.

Reply:
Good catch, knew I was forgetting something!